src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -10,6 +10,11 @@
 using System.Reflection;
 using System.Threading;
 
+[assembly: UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
FYI @vitek-karas @mateoatr @tlakollo - this was the only way I figured out how to suppress this warning in code. The same thing didn't work when I applied this attribute to the `TypeDescriptorComObject` class. Is that expected?